util/src/main/scala/scala/scalanative/util/package.scala

```diff
@@ -54,4 +54,8 @@ package object util {
   def partitionBy[T](elems: Seq[T], batches: Int)(
       f: T => Any): Map[Int, Seq[T]] =
     elems.groupBy { elem => Math.abs(f(elem).##) % batches }
+
+  def parallelStream[T](elems: Iterable[T]): JStream[T] = {
+    JStream.of(elems.toSeq: _*).parallel()
+  }
```

Comment:
Consider not expanding the API of `util`, and instead explicitly do this at call site. It's not big enough to be worth it.